src/main/java/tutorlink/resultpackage/CommandResult.java

```diff
@@ -0,0 +1,26 @@
+package tutorlink.resultpackage;
+
+import tutorlink.listpackage.ItemList;
+
+public class CommandResult {
+
+    private static final String LINE_SEPARATOR = "\n\t";
+
+    public final String message;
+    public final ItemList returnedList;
+
+    public CommandResult(String message) {
+        this.message = message;
+        this.returnedList = null;
+    }
+
+    public CommandResult(String message, ItemList returnedList) {
+        this.message = message;
+        this.returnedList = returnedList;
+    }
+
+    @Override
+    public String toString() {
```

Comment:
I like the toString() command is written in a clear and concise manner that is easy to understand.